cli/command/formatter/sorter.go

```diff
@@ -0,0 +1,321 @@
+package formatter
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+	"time"
+)
+
+const (
+	ascendantKey  = "asc"
+	descendantKey = "desc"
+)
+
+type fieldIndex []int
+
+type orderBy struct {
+	name     string     // order by name
+	ordering string     // asc or desc
```

Comment:
Would it be weird to just make it a boolean, `asc`, that if set to `false` would sort descending?